docs/playbooks/alerts/StackdriverExportFailed.md

```diff
@@ -0,0 +1,29 @@
+# StackdriverExportFailed
+
+OpenCensus failed to export metrics to Stackdriver.
+
+This means all other alerts we have configured won't work as the alert depend
```

Comment:
This sounds unnecessarily alarmist. There are many reasons it could fail, including the container is shutting down.

Reply:
Indeed. Let me update this playbook.